build.sbt

```diff
@@ -131,3 +135,6 @@ lazy val root = (project in file("."))
 
 Test / fork := true
 Test / javaOptions += "-Dconfig.file=src/test/resources/application-test.conf"
+
+IntegrationTest / fork := true
+IntegrationTest / javaOptions += "-Dconfig.file=src/it/resources/application-it.conf"
```

Comment:
I would namespace this `Test /` and `IntegrationTest /` setting under the specific project that requires it

Reply:
done! 👍